### DISCUSSION TITLE
where to configure the jellyfin part ( authelia oidc login )

### DISCUSSION THREAD

**Discussion by loeken** [9p4/jellyfin-plugin-sso#172] [Q&A] [answered]
https://github.com/9p4/jellyfin-plugin-sso/blob/main/providers.md#jellyfins-config

i followed the instructions to setup jellyfin ( i run jellyfin via helm chart in a container ) but i dont see where this configuration would end up in ( my jellyfin install doesnt have any yaml based config )

could somebody clarify where this configuration block should end up? are those some settings to be done in the ui ?

**Comment by 9p4** [comment]
The YAML is just meant as a human-readable text-based configuration reference. The plugin can be configured either through the API or the web interface.

**Comment by loeken** [comment]
hmm now i am even more confused, i enabled the repo, installed the plugin but when i click on it in the ui it says "This plugin has no settings to set up." when i hit the api url i get a 404. im using jellyfin 10.8.10 do you have any clue what im doing wrong here :D ?

**Reply by 9p4** [reply]
Strange. What repo are you using?

**Reply by loeken** [reply]
https://raw.githubusercontent.com/9p4/jellyfin-plugin-sso/manifest-release/manifest.json

**Reply by loeken** [reply]
I build a new docker image this morning for jellyfin 10.8.13 and now i can reach the configuration part

**Reply by loeken** [reply]
okey configured it, all working great with authelia for me thanks for the plugin

**Comment by loeken** [comment, marked as answer]
works now with my image loeken/jellyfin:10.8.13
